hedera-node/test-clients/src/main/resource/contract/contracts/Create2FactoryWithSelfDestructingContract/Create2FactoryWithSelfDestructingContract.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+contract Create2FactoryWithSelfDestructingContract {
+    event Deployed(address addr, uint salt);
+
+    // 1. Get bytecode of contract to be deployed
+    // NOTE: _owner and _foo are arguments of the SelfDestructingTestContract's constructor
+    function getBytecode(address _owner, uint _foo) public pure returns (bytes memory) {
```

Comment:
Nit:  I understand _foo is just a random uint but maybe a better name would be good?